Support/Testing/CircleCI/Dockerfile

```diff
@@ -40,6 +40,35 @@ MAINTAINER Stephane Sezer <sas@fb.com>
 RUN apt-get update
 RUN apt-get install -y software-properties-common python-software-properties wget
 
+# Incase lldb needs built
+RUN apt-get install -y libz-dev
+RUN apt-get install -y swig
+RUN apt-get install -y ncurses-dev
+
+# Make the developer's life not suck
+RUN apt-get -y install sudo
+RUN apt-get -y install zsh
+RUN apt-get -y install tmux
+RUN apt-get -y install curl
+RUN apt-get -y install vim
+RUN apt-add-repository -y ppa:neovim-ppa/stable
+RUN apt-get update -y
+RUN apt-get -y install neovim
+
+# Debugging tools
+RUN apt-get install -y strace
+RUN apt-get install -y htop
+RUN apt-get install -y psmisc
+
+# Python 3.7 and misc
+RUN add-apt-repository -y ppa:deadsnakes/ppa
+RUN apt-get update -y
+RUN apt-get install -y python3.7
+RUN curl https://bootstrap.pypa.io/get-pip.py | python3.7
```

Comment:
Is this required? Isn't there just a `pip` package in apt, maybe from the deadsnakes repo?

Reply:
`apt install python3-pip` installs to `/usr/local/lib/python3.5/dist-packages`